dev/com.ibm.ws.jdbc_fat/publish/servers/com.ibm.ws.jdbc.fat/server.xml

```diff
@@ -261,6 +261,10 @@
 	<javaPermission className="java.lang.RuntimePermission" name="accessDeclaredMembers" />
 	<!-- Postgres and SQLServer permission to make a basic query-->
 	<javaPermission className="java.util.PropertyPermission" name="*" actions="read"/>
+
+	<!-- Postgres permission during commit cycle -->
+	<javaPermission className="org.osgi.framework.ServicePermission" name="*" actions="get"/>
+	<javaPermission className="javax.security.auth.PrivateCredentialPermission"  name='* * "*"' actions="read" />
```

Comment:
does `name="*"` not work here?

Reply:
I don't believe so.  I can run that test tomorrow. I tried various combinations and nothing worked.  I then did some looking in other places and found that line.  I copied it over and that worked.

Reply:
If it works we can leave it.  But from my understanding only the last * in a java permission statement actually does anything.

Reply:
I changed that javaPermission line to name="*" and re-ran the FAT test bucket and it failed with java 2 security errors.  I changed it back to what I have above and the FAT test bucket passed.